ui/src/config.js

```diff
@@ -1 +1,3 @@
-export const API_BASE_URL = "http://localhost:3001"
\ No newline at end of file
+export const API_BASE_URL   = "http://localhost:3001"
```

Comment:
this is great - good that you are defining constants like this in a separate file